Tasks/Common/webdeployment-common-v2/ziputility.ts

```diff
@@ -124,9 +123,8 @@ export async function unzip(zipFileLocation: string, unzipDirLocation: string) {
     tl.debug('windows platform: ' + isWin);
 
     tl.debug('extracting ' + zipFileLocation + ' to ' + unzipDirLocation);    
-    if (isWin) {
-        let isMSBuildPackage = await isMSBuildPackageZip(zipFileLocation);
```

Comment:
Consider removing the function `isMSBuildPackageZip ` as it is not getting used elsewhere.